plugins/woocommerce-blocks/docs/third-party-developers/extensibility/data-store/cart.md

```diff
@@ -551,6 +552,21 @@ const { dispatch } = useDispatch( CART_STORE_KEY );
 dispatch( updateCustomerData( customerData, editing ) );
 ```
 
+### applyingExtensionCartUpdate
```

Comment:
The section `applyingExtensionCartUpdate` is not listed on the TOC. It might be worth updating the TOC in general.  There might be more sections missing.

Apart from that, what are your thoughts on placing the section `applyingExtensionCartUpdate` below the section `applyExtensionCartUpdate`?